x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -384,7 +383,7 @@ public class Constants {
         "cluster:monitor/xpack/usage/analytics",
         "cluster:monitor/xpack/usage/archive",
         "cluster:monitor/xpack/usage/ccr",
-        DataStreamLifecycle.isFeatureEnabled() ? "cluster:monitor/xpack/usage/data_lifecycle" : null,
```

Comment:
This is needed for release-tests to pass (#97442 removed the feature flag check for the action). I could have also muted it. Since the fix is so simple, I just fixed it here.